easybuild/easyconfigs/t/tagdust/tagdust-2.33-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,34 @@
+# easybuild easyconfig
+#
+# Fred Hutchinson Cancer Research Center - Seattle Washington - US
+# John Dey <jfdey@fredhutch.org>
+#
+easyblock = 'ConfigureMake'
+
+name = 'tagdust'
```

Comment:
@fizwit We try to use "official" software names, based on the contents of http://tagdust.sourceforge.net this should be changed to `TagDust`?

Reply:
done in d6c7c8c